frontend/src/board/pgn/boardTools/underboard/settings/GameSettings.tsx

```diff
@@ -32,6 +32,8 @@ import { useNavigate } from 'react-router-dom';
 import { useChess } from '../../../PgnBoard';
 import AnnotationWarnings from '../../../annotations/AnnotationWarnings';
 import RequestReviewDialog from './RequestReviewDialog';
+import { UnsavedGameBanner } from '@/components/games/edit/UnsavedGameBanner';
```

Comment:
Are these changes necessary? There are no other changes in the file.

Reply:
There should be. Something went awry here, the game settings should show a non-dismissible unsaved banner.

Reply:
It looks like you put it in EditorSettings instead, which probably isn't the correct place for it